apps/web/src/components/status-bar/index.tsx

```diff
@@ -160,6 +165,28 @@ function StatusBar() {
               </Text>
             </Button>
           )}
+          {doesVaultExist && (
+            <Button
+              variant="statusitem"
+              onClick={isVaultLocked ? unlockVault : lockVault}
+              sx={{
+                alignItems: "center",
+                justifyContent: "center",
+                display: "flex"
+              }}
+            >
+              {isVaultLocked ? (
+                <Lock size={10} color="var(--icon-error)" />
+              ) : (
+                <Unlock size={10} color="var(--icon-success)" />
```

Comment:
Just use the default icon color for both. It'll be distracting otherwise. Also, I don't think `var(--` is necessary. Direct color names will work too.

Reply:
I'm not fond of the color as well. I put it because at that icon size it's hard to tell between the lock and unlock icon. I've removed the colors, if you have any suggestion here, please let me know